glean-core/android/src/test/java/mozilla/telemetry/glean/TestUtil.kt

```diff
@@ -236,3 +238,19 @@ internal fun getMockWebServer(): MockWebServer {
     })
     return server
 }
+
+/**
+ * Convenience method to get the body of a request as a String.
+ * If the request body is gzipped, it will be decompressed. Otherwise
+ * the UTF8 representation will be reported.
```

Comment:
This reads as if the decopmressed data isn't UTF-8.

How about:

```suggestion
 * The UTF8 representation of the request body will be returned.
 * If the request body is gzipped, it will be decompressed first. 
```